server/src/app.js

```diff
@@ -153,14 +153,14 @@ class Events extends EventEmitter {
       if (client) {
         flightId = client.flightId;
       }
-      const event = {
-        event: eventName,
-        params: param,
-        clientId: clientId,
-        flightId: flightId,
-        timestamp: new Date()
-      };
-      this.events.push(event);
+      // const event = {
```

Comment:
Why are we commenting out a block of code and leaving it here?

Reply:
Coz I'm a lazy cuss.

This code effectively does nothing. It stores each event that happens while Thorium is running, but there is no mechanism for accessing those events. The hope is that eventually someone will come up with a clever way of doing analytics on those events, but until that happens, storing each event just increases the load on memory. This should improve some of the memory-leak problems that the CMSC ships have been experiencing.